content/js1/sprints/1/prep/index.md

```diff
@@ -8,4 +8,443 @@ backlog= 'Module-JS1'
+
+So this new problem can be expressed as a question:
+
+> ❓ Problem
+>
+> "How can we check the values of variables in our program when it is being executed?"
+
+### 🖨️ Printing to the terminal
+
+In order to check the values when our program runs, we can make use of a function called `console.log`.  
+`console.log` will allow us to log expressions to the console while our program is executing. This is very useful for complex programs when we need to check the value of expressions in our program.
+
+Let's see how to use `console.log`
+
+In a file called `example.js`, we can write name of the function `console.log`.
+
+```js title="example.js"
+console.log;
+```
+
+However, if we run this file with Node, we won't be able to see anything in the terminal.
+As with `Math.round` we need to use the syntax for calling a function.
+So we can add parentheses after the function name to indicate we're calling this function:
+
+```js title="example.js"
+console.log("hello there!");
+```
+
+We should see the string "hello there!" logged out in the terminal.
+
+### Saving function output
+
+We can use variables to store the output from a function.
+
+Let’s consider the following file, `arithmetic.js`;
+
+```js title="arithmetic.js"
+const result = Math.round(10.3);
+```
+
+When this program is executed, it creates a variable called result and assigns to it **the output of the function**, in this case the rounded number.
+
+So `result` will store the value `10`.
+
+### 🔭 Logging vs function output
+
+Some functions output values that we can use in our program.
+
+`Math.round` takes an input, does a calculation and then outputs values that we can use when our program is running.
+
+However, some functions don't produce useful outputs in our running program; however they can still cause **effects**. Consider the following line:
+
+```js
+const result = console.log("hello world");
+```
+
+{{<note title="Exercise" type="exercise">}}
+
+Try predicting what `result` will evaluate to when the code above runs. Try executing this line in the Node REPL to observe what happens.
+
+{{</note>}}
+
+When this program runs, the variable `result` will evaluate to `undefined`.
+
+{{<note title="Note" type="note">}}
+
+`undefined` is a data type in JavaScript often used to represent the absence of a specific value
+
+{{</note>}}
+
+This is potentially confusing as `console.log` is a function with a set of instructions. `console.log` _does_ have an effect: it logs values to the console. However, `console.log` doesn't produce an output that we can use _inside_ the rest of our running program
+
+{{<note type="info">}}
+
+Key fact: `console.log` is used to print values to the terminal. It doesn’t produce an output in the running program.
+
+{{</note>}}
```

Comment:
I feel like this ends very bluntly; I'd be tempted to add a wrap-up/recap section reprising the learning objectives here.